src/v2/wallet.ts

```diff
@@ -0,0 +1,1980 @@
+import common = require('../common');
```

Comment:
move to modules/core/src/v2

Reply:
yea, deleted this file completely, not sure what happened here, probably a bash stash